base/main_test_bucket_pool_config.go

```diff
@@ -81,6 +81,25 @@ func TestsRequireMobileRBAC(t *testing.T) {
 	}
 }
 
+// canUseMobileXDCR checks if cluster supports mobile XDCR
+func (tbp *TestBucketPool) canUseMobileXDCR(ctx context.Context) (bool, error) {
+	var clusterSupport bool
+	if tbp.cluster != nil {
```

Comment:
When is this nil?

Reply:
Don't think it ever should be but did this to be panic safe